### PR TITLE
{CI} Update `[Microsoft Internal / Service Team] Feature Request` template

### PR DESCRIPTION
**Pull request by wangzelin007**
- If user creates a feature request with the template(`[Microsoft Internal / Service Team] Feature Request`), automatically add the label `feature-request`.
- Modify `Swagger PR link` to `Swagger PR link / SDK link`.
- Add `If you cannot provide, please fill in N/A.` to description.

### CODE REVIEW

**Review by zhoxing-ms — APPROVED** [review]

**Comment by yonzhan** [comment]
CI

**Review by yonzhan — APPROVED** [review]

**Review by evelyn-ys — APPROVED** [review]

**Review by necusjz — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/Service_team_request.yaml [hunk 1]
```diff
@@ -19,15 +20,19 @@ body:
...
       placeholder: e.g., Microsoft.Compute/disks
     validations:
       required: true
   - type: textarea
     id: feature-desc
     attributes:
       label: Description of Feature or Work Requested
-      description: Provide a brief description of the feature or problem. A link to conceptual documentation may be helpful too.
+      description: |
+        Provide a brief description of the feature or problem. A link to conceptual documentation may be helpful too.
```
Comment (jsntcy) [inline .github/ISSUE_TEMPLATE/Service_team_request.yaml:34]: Should we remove "problem" if this template is only for feature request?

File: .github/ISSUE_TEMPLATE/Service_team_request.yaml [hunk 2]
```diff
@@ -36,15 +41,19 @@ body:
     id: mini-api
     attributes:
       label: Minimum API Version Required
-      description: What is the minimum API version of your service required to implement your feature?
+      description: |
+        What is the minimum API version of your service required to implement your feature?
```
Comment (jsntcy) [inline .github/ISSUE_TEMPLATE/Service_team_request.yaml:45]:
```suggestion
        What is the minimum API version of your service required to support your feature?
```

File: .github/ISSUE_TEMPLATE/Service_team_request.yaml [hunk 3]
```diff
@@ -36,15 +41,19 @@ body:
...
     validations:
       required: true
   - type: textarea
     id: swagger
     attributes:
-      label: Swagger PR link
-      description: Provide a link to the location of your feature(s) in the REST API specs repo. If your feature(s) has corresponding commit or pull request in the REST API specs repo, provide them. This should be on the master branch of the REST API specs repo.
+      label: Swagger PR link / SDK link
+      description: |
+        Provide a link to the location of your feature(s) in the REST API specs repo. If your feature(s) has corresponding commit or pull request in the REST API specs repo, provide them. This should be on the master branch of the REST API specs repo.
```
Comment (jsntcy) [inline .github/ISSUE_TEMPLATE/Service_team_request.yaml:55]: Should be "main" branch instead of "master" branch as "master" was already deprecated.

Comment (jsntcy) [inline .github/ISSUE_TEMPLATE/Service_team_request.yaml:55]: Better to provide link to swagger repo, such as [swagger repository](https://github.com/Azure/azure-rest-api-specs)

Comment (wangzelin007) [reply .github/ISSUE_TEMPLATE/Service_team_request.yaml:55]: Already provide a placeholder: e.g., https://github.com/Azure/azure-rest-api-specs/pull/16629
image

**Pull request merged**